### ISSUE TITLE
Is it really unsupervised model?

### ISSUE THREAD

**Issue by JawadTawhidi** [suhwan-cho/TMO#5]
Hi, I am new in the area of Unsupervised video object segmentation, I have read your paper and implemented your pretrained model. 

My question is that is your model for unsupervised video object segmentation, if so, why it is taking ground truth mask in the training and testing stages? As I know in unsupervised methods there is no need for providing masks for the model at the training and testing stages of a the model.

**Comment by suhwan-cho** [comment]
Hi Jawad,

In VOS field, the word "unsupervised" is usually equal to "zero-shot". It means that there is no guidance on the object to segment during inference. In this case, "unsupervised" has nothing to do with network training protocol. During testing, TMO does not use ground truth masks for inference. They are just loaded for performance evaluation.

**Comment by JawadTawhidi** [comment]
Thank you for your response, I checked your codes again.

Now I think you did not use the masks during both training and testing
stages of your model, am I correct?

**Comment by JawadTawhidi** [comment]
Thank you so much for your response.
